frontend/public/components/utils/build-strategy.tsx

```diff
@@ -27,6 +33,8 @@ export const BuildStrategy: React.SFC<BuildStrategyProps> = ({ resource, childre
     {git && <dd>{git.uri}</dd>}
     {gitRef && <dt>Git Ref</dt>}
     {gitRef && <dd>{gitRef}</dd>}
+    {commitMessage && <dt>Git Commit</dt>}
+    {commitMessage && <dd>{commitMessage} {commitHash && <code>{commitHash}</code>} {commitAuthorName && `authored by ${commitAuthorName}`}</dd>}
```

Comment:
I know we put these together in OpenShift console, but should each of these commit properties just be separate dt/dd elements? I think it would be easier to understand and would avoid problems with long values wrapping

Reply:
The downside is now we get unbalanced columns.  We move items between columns?

<img width="1404" alt="screen shot 2018-07-09 at 4 36 43 pm" src="https://user-images.githubusercontent.com/895728/42474671-56e5fd76-8396-11e8-99ee-d36962004b8a.PNG">

Reply:
OK, seeing it here, I'm fine with the before. Thanks.

We might want to shorten the hash to 7 chararcters.

Throwing this out as an option as well: maybe make it two lines?

My commit message here
`23e7e68` by Ben Parees

(I'm fine whatever way we go.)

Reply:
I like the two line suggestion.  Thanks! 
![screen shot 2018-07-10 at 8 35 17 am](https://user-images.githubusercontent.com/895728/42510216-3f2bfccc-841c-11e8-9c3c-2b8f0a9da2bd.PNG)